src/main/java/org/jboss/aerogear/unifiedpush/message/UnifiedMessage.java

```diff
@@ -248,6 +255,7 @@ public MessageBuilder(Builder builder) {
 
         private final Builder builder;
         private final Message message = new Message();
+        private  WindowsBuilder windowsBuilder;
```

Comment:
ah, makes sense - but was not obvious

On Mon, Mar 9, 2015 at 6:39 PM, Sebastien Blanc <notifications@github.com>
wrote:

> In
> src/main/java/org/jboss/aerogear/unifiedpush/message/UnifiedMessage.java
> <https://github.com/aerogear/aerogear-unifiedpush-java-client/pull/68#discussion_r26057434>
> :
>
> > @@ -248,6 +255,7 @@ public MessageBuilder(Builder builder) {
> >
> >          private final Builder builder;
> >          private final Message message = new Message();
> > +        private  WindowsBuilder windowsBuilder;
>
> Because of this method
> https://github.com/aerogear/aerogear-unifiedpush-java-client/pull/68/files#diff-63e90d62fb5413ab881ba56b0eb54a32R373
> , so we only initialize the windowsBuilder if needed but I can change that,
> make it final and always initialize it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aerogear/aerogear-unifiedpush-java-client/pull/68/files#r26057434>
> .
>



-- 
Matthias Wessendorf

blog: http://matthiaswessendorf.wordpress.com/
sessions: http://www.slideshare.net/mwessendorf
twitter: http://twitter.com/mwessendorf